events/message.js

```diff
@@ -1,20 +1,23 @@
 const { client } = require("../index.js");
 module.exports = {
-  name: "message",
-  once: false,
-  execute: async (message) => {
-    prefix = "t!" || "t~";
+    name: "message",
+    once: false,
+    execute: async (message) => {
+        prefix = "t!" || "t~";
 
-    const args = message.content.slice(prefix.length).trim().split(/ +/g);
+        const args = msg.startsWith(prefix)
+            ? msg.slice(prefix.prefix).trim().split(/ +/g)
```

Comment:
`prefix` is a string. I don't think `prefix.prefix` is a thing.

Reply:
ya typing mistake is should have been length